include/ring-core/base.h

```diff
@@ -89,6 +89,14 @@
 #elif defined(__MIPSEL__) && defined(__LP64__)
 #define OPENSSL_64_BIT
 #define OPENSSL_MIPS64
+#elif defined(__riscv) && __riscv_xlen == 64
+#define OPENSSL_64_BIT
+#define OPENSSL_RISCV64
+#define OPENSSL_NO_ASM
+#elif defined(__riscv) && __riscv_xlen == 32
+#define OPENSSL_32_BIT
+#define OPENSSL_RISCV32
+#define OPENSSL_NO_ASM
```

Comment:
Ditto.

Reply:
Is there a real-world need for the 32-bit RISC-V? The PR adds the 64-bit stuff to the GitHub Actions matrix but not the 32-bit. 

I noticed BoringSSL doesn't define OPENSSL_RISCV32.

Reply:
Same as above, removed in 71dc9e9